lib/models/EMailer.js

```diff
@@ -24,4 +24,16 @@ export default class EMailer {
   static unknown() {
     return new EMailer();
   }
+
+  /**
+   * Creates a _EMailer_ from its JSON representation.
+   *
+   * @param jmap {Client} The {@link Client} instance that originates the _EMailer_ instance. Is actually ignored.
+   * @param object {Object} The JSON representation of the _EMailer_, as a Javascript object
+   *
+   * @return {EMailer}
+   */
+  static fromJSONObject(jmap, object) {
```

Comment:
Because it's an API method, used throughout the library. I kept the signature of the method.